pkgs/development/interpreters/trealla/default.nix

```
@@ -27,7 +27,7 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ xxd ];
   buildInputs = [ readline openssl libffi ];
-  checkInputs = [ valgrind ];
+  checkInputs = lib.optionals (!stdenv.isDarwin) [ valgrind ];
```

Valgrind builds still work on [x86_64-darwin](https://hydra.nixos.org/job/nixos/trunk-combined/nixpkgs.valgrind.x86_64-darwin) so this should instead be:

```suggestion
  checkInputs = lib.optionals (!(stdenv.isDarwin && stdenv.isAarch64)) [ valgrind ];
```